app/uk/gov/hmrc/gform/testonly/TestOnlyController.scala

```diff
@@ -528,18 +528,33 @@ class TestOnlyController(
   }
 
   def restoreAll(formTemplateId: FormTemplateId, maybeAccessCode: Option[AccessCode]) =
+    controllerComponents.actionBuilder.async { implicit request =>
+      val snapshotId = request.body.asFormUrlEncoded.get("snapshotId").head
```

Comment:
`.head` is unsafe, can we raise a better error when there will not be a `snapshotId`?

Reply:
oops
that was temporary for experimentation, but I forgot to change it back :(